### PR TITLE
Plan for CDI 4.1 release

### PR DESCRIPTION
**Pull request by starksm64**
## Specification PR template
When creating a specification project release review, create PRs with the content defined as follows.

Include the following in the PR:
- [x] A directory in the form wombat/x.y where x.y is the release major.minor version, and the directory contains the following.
- [ ] Specification PDF in the form of jakarta-wombat-spec-x.y.pdf
- [ ] Specification HTML in the form of jakarta-wombat-spec-x.y.html
- [x] A specification page named _index.md following the template at:
      https://github.com/jakartaee/specification-committee/blob/master/spec_page_template.md
- [ ] For a Progress Review, that sufficient progress has been made on a Compatible Implementation and TCK, to ensure that the spec is implementable and testable.
- [ ] For a [Release Review](https://www.eclipse.org/projects/handbook/#release-review), a summary that a Compatible Implementation is complete, passes the TCK, and that the TCK includes sufficient coverage of the specification. The TCK users guide MUST include the instructions to run the compatible implementations used to validate the release.
Instructions MAY be by reference.
   - [ ] Updated release record
   - [ ] Generated IP Log
   - [ ] Email to PMC
   - [ ] Start release review by emailing EMO 
- [ ] The URL of the OSSRH staging repository for the api, javadoc:
      <add URL here>
- [ ] The URL of the staging directory on downloads.eclipse.org for the proposed EFTL TCK binary:
      <add URL here>
- [ ] The URL of the compatibility certification request issue:
      <add URL here>
- [ ] Specification JavaDoc in the wombat/x.y/apidocs directory. 
If desired, an optional second PR can be created to contain just the JavaDoc in the `apidocs` directory.

Note: If any item does not apply, check it and mark N/A below it.

### CODE REVIEW

File: cdi/4.1/_index.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+Jakarta Contexts Dependency Injection specifies a means for obtaining objects in such a way as to maximize reusability, testability and maintainability compared to traditional approaches such as constructors, factories, and service locators (e.g., JNDI).
+
+### New features, enhancements or additions
+<!-- List here -->
+Several minor issues in the Specification, APIs and TCK will be addresses as detailed in the [CDI 4.1 Milestone](https://github.com/jakartaee/cdi/milestone/1)
+
+### Removals, deprecations or backwards incompatible changes
+<!-- List here -->
+* The Expression Language SPI methods on the BeanManager will be deprecated in preparation for a move to a new subinterface
+that will be part of a new artifact. This removes the dependency from the Jakarta Expression Language core API, bot the
```
Comment (Emily-Jiang) [inline cdi/4.1/_index.md:16]:
```suggestion
that will be part of a new artifact. This removes the dependency from the Jakarta Expression Language core API, but the
```

File: cdi/4.1/_index.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+clean-up dependencies and refactor features that are not client facing.
```
Comment (Emily-Jiang) [inline cdi/4.1/_index.md:17]:
```suggestion
clean-up dependencies and refactor features are not client facing.
```

Comment (starksm64) [reply cdi/4.1/_index.md]: That whole sentence was awkward, so I have reworded it.

**Comment by dblevins** [comment]
@starksm64 could we put a small executive summary of the more interesting changes?  Ok to have a link to the project board, but unless we record exactly what items were there at the time of the vote, there's no record of exactly what technical direction was approved.

**Comment by Emily-Jiang** [comment]
@starksm64 can you please fix the ECA failures?

**Comment by Emily-Jiang** [comment]
Based on what @dblevins asked and you agreed @starksm64, I won't merge this PR even though the ballot has completed successfully. I will put the ballot result on hold until you have resolved this and ready to proceed with the next step.

**Comment by ivargrimstad** [comment]
Oh, I missed that last comment. Sorry about that!
Let's add the executive summary as a separate PR to avoid the hassle of reverting the PR.

**Comment by starksm64** [comment]
Commit 1468ea1c504dafd862f95c012fdb62181874223d did add the summary so this should be fine

**Pull request merged**